steemrocks/models.py

```diff
@@ -485,6 +487,28 @@ def received_sp(self):
     def total_sp(self):
         return round(self.sp + self.received_sp - self.delegated_sp, 2)
 
+    @property
+    def worth_sp(self):
+        s = Steem()
```

Comment:
can we use get_steem_conn() (located in in utils.py) to get a Steem instance?

Reply:
Sure!